client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfiguration.java

```diff
@@ -23,6 +23,7 @@
 import com.palantir.conjure.java.api.config.service.PartialServiceConfiguration;
 import com.palantir.conjure.java.api.config.service.ServiceConfiguration;
 import com.palantir.conjure.java.api.config.service.UserAgent;
+import com.palantir.conjure.java.okhttp.HostEventsSink;
```

Comment:
okhttp :-/

Reply:
Yep. This is gross. Unfortunately I think changing this package name is gonna be a whole _world_ of pain.

Reply:
How terrible would it be to redefine in a new package and update the old interfaces to extend the new?

Reply:
Hmm ok I think we might actually be able to do that - let me give it a shot